### ISSUE TITLE
Add a `Debug` menu drop down to CoqIDE

### ISSUE THREAD

**Issue by Alizter** [coq/coq#14909]
This is a feature request in relation to #14644. I think it would be better to have a drop down menu titled `Debug` or `Debugging`. I don't think the current `View` menu is entirely appropriate for debugging functions. Not to mention, `View` has become a bit overcrowded.

I think `Debug` or `Debugging` should lie between the `Compile` and `Windows` menu items.

cc @jfehrle

**Comment by Alizter** [comment]
cc @jfehrle

**Comment by jfehrle** [comment]
Yes, I saw this when you submitted the issue and I expect to change it.  Are you suggesting that it has some urgency?

If you've been using the debugger, I'd like to hear your impressions of it.  Probably not here, though.

**Comment by Alizter** [comment]
@jfehrle No urgency, I wasn't sure if you saw it that's all.
